### DISCUSSION TITLE
Quick note of congrats and better than supabase

### DISCUSSION THREAD

**Discussion by osseonews** [nhost/nhost#2643] [General]
Been following Nhost for awhile, as we have been using self-hosting Hasura since the start of Hasura and also sometimes Supabase. Well after you launched AI recently, I decided to test again on Nhost and was blown away by the progress you have made in everything. We finally signed up to Pro. Honestly, your entire platform is incredible and much better than Supabase in many respects (Hasura is way better than Postgrest, which is what Supbase is built on). So honestly, you guys should be bigger than Supabase, or at least just as popular. I think this is just a marketing thing really and maybe you need more investors. Hope this message helps. Keep up the amazing work.

**Comment by dbarrosop** [comment]
Thanks, we really appreciate the kind words :)
